airbyte-webapp/src/components/EntityTable/components/StatusCell.tsx

```diff
@@ -30,8 +30,7 @@ const StatusCell: React.FC<IProps> = ({ enabled, isManual, id, onChangeStatus, i
   }, []);
 
   if (!isManual) {
-    const onSwitchChange = (event: React.SyntheticEvent) => {
-      event.stopPropagation();
```

Comment:
Because the switch button is behind a clickable row, this causes the row to trigger the navigation to the detail page and also the switch. This is why stopPropagation was set here. Unfortunately, we don't seem to have access to the event object in Headless UI, so we'd have to work around another way to make it work.